src/auth/components/RegisterForm/style.less

```diff
@@ -40,16 +41,21 @@
           border: none;
           flex-grow: 1;
           font-size: 16px;
+          background-color: var(--theme-background);
+          color: var(--theme-text-on-background-1);
```

Comment:
is this needed? (see above comment)

Reply:
Title not required. Required to match background in dark mode.